cumulusci/tasks/push/tests/test_push_tasks.py

```diff
@@ -448,9 +448,7 @@ def test_schedule_push_org_query_get_org_error():
         options={
             "orgs": ORG,
             "version": VERSION,
-            "namespace": NAMESPACE,
```

Comment:
Is this change intended to be in this PR?

Reply:
Yeah I figured I'd increase code coverage while were at it.